package.json

```diff
@@ -6,6 +6,8 @@
     "@craco/craco": "^6.4.0",
     "@headlessui/react": "^1.4.1",
     "@heroicons/react": "^1.0.5",
+    "@material-ui/core": "^4.12.3",
+    "@material-ui/icons": "^4.11.2",
```

Comment:
don't use material UI, we already agreed on tailwindCSS, this need a lot of customization